engine/order_manager_test.go

```diff
@@ -27,6 +30,17 @@ type omfExchange struct {
 	exchange.IBotExchange
 }
 
+var btcusdPair currency.Pair
+
+func TestMain(m *testing.M) {
+	var err error
+	btcusdPair, err = currency.NewPairFromString("BTCUSD")
+	if err != nil {
+		log.Fatal(err)
+	}
+	os.Exit(m.Run())
+}
+
```

Comment:
```suggestion
var btcusdPair = currency.NewPair(currency.BTC, currency.USD)

```

Reply:
Thanks. Will DriveBy this somewhere else too then.